code/dijkstra-algorithm/Infrastructure/src/main/java/ru/unn/agile/dijkstraalgorithm/infrastructure/TextLogger.java

```diff
@@ -0,0 +1,69 @@
+package ru.unn.agile.dijkstraalgorithm.infrastructure;
+
+import ru.unn.agile.dijkstraalgorithm.viewmodel.ILogger;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+import java.util.Locale;
+
+public class TextLogger implements ILogger {
+    private final String filename;
+    private final BufferedWriter writer;
+    private static final String DATE_FORMAT_POINT = "yyyy-MM-dd HH:mm:ss";
```

Comment:
yyyy-MM-dd HH:mm:ss.SSS should be more usefull

Reply:
It is!